clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/ResizeServiceAtomicOperation.java

```diff
@@ -53,27 +55,68 @@ private static Task getTask() {
   }
   @Override
   public Void operate(List priorOutputs) {
-    getTask().updateStatus(BASE_PHASE, "Initializing Resize ECS Server Group Operation...");
+    updateTaskStatus("Initializing Resize ECS Server Group Operation...");
 
-    String serverGroupName = description.getServerGroupName();
-    Integer newSize = description.getCapacity().getDesired();
+    resizeService();
```

Comment:
Is it necessary to rescale the service if you resize the auto scaling group?  I'm sincerely asking

Reply:
It is. We are using the AutoScaling group as a place to store the data related to how many instances we want when a service is enabled. Instead of just assuming that it is 1. 

Since we are not attaching an Scaling Policy. ECS will not know when to scale up or down a cluster.